README.md

```diff
@@ -1 +1,24 @@
 # Slurm rocks
+
+The `slurm-rocks` repository is a collection of files to build rocks for the upstream
+Slurm services.
+
+## Getting started
+
+To get started with the project, you must have [Rockcraft] and [Just] installed for
+packing the rocks, [Skopeo] for copying the generated OCI images to your local Docker registry, and
+finally [Docker] for running the images.
+
+```bash
+just pack # Packs all the rocks.
+just import # Imports all the rocks into your local Docker registry.
+docker run --rm -d --name slurmctld slurmctld:latest # Runs the slurmctld rock with Docker.
```

Comment:
[nit]: Typically for inline comments I add two spaces instead of one. It's a Pythonism, but I find it's helpful for visually separating the comment from the command:

```suggestion
just pack  # Packs all the rocks.
just import  # Imports all the rocks into your local Docker registry.
docker run --rm -d --name slurmctld slurmctld:latest  # Runs the slurmctld rock with Docker.
```

Reply:
Fixed